atomicClient.html

```diff
@@ -0,0 +1,249 @@
+<head>
```

Comment:
This code should be put into tests directories, since we are not intending on shipping this.